sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div class="m-3">
+    <v-row>
+      <header-title-value
+        title="Submitted date"
+        :value="
+          headerDetails.submittedDate
+            ? dateOnlyLongString(headerDetails.submittedDate)
+            : '-'
+        "
+      />
+      <Separator v-if="headerDetails.institutionName"></Separator>
+      <header-title-value title="Institution"
+        ><template #value
+          ><span class="link-primary" @click="navigateToInstitutionProfile()">
+            {{ headerDetails.institutionName }}
+          </span>
+        </template></header-title-value
+      >
+      <Separator v-if="headerDetails.locationName" />
+      <header-title-value
+        v-if="headerDetails.locationName"
+        title="Location"
+        :value="headerDetails.locationName"
+      />
+    </v-row>
+    <v-row class="mt-4" v-if="showReviewDetails">
+      <header-title-value
+        :iconLabel="iconLabel"
+        :iconColor="iconColor"
+        :title="assessedByLabel"
+        :value="assessedByLabelValue"
+      />
+      <Separator />
+      <header-title-value
+        :title="assessedDateLabel"
+        :value="assessedDateLabelValue"
+      />
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  ApplicationOfferingChangeRequestHeader,
+  ApplicationOfferingChangeRequestStatus,
+} from "@/types";
+import { useFormatters } from "@/composables";
+import { computed, defineComponent, watchEffect, ref } from "vue";
+import { useRouter } from "vue-router";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+
+interface ReviewLabels {
+  assessedByLabel: string;
+  assessedDateLabel: string;
+}
+
+export default defineComponent({
+  props: {
+    headerDetails: {
+      type: Object,
```

Comment:
I believe that line 62 must be declared as below, otherwise, it will be shown as `any`.
```ts
props: {
    headerDetails: {
      type: Object as PropType<ApplicationOfferingChangeRequestHeader>,
      required: true,
      default: {} as ApplicationOfferingChangeRequestHeader,
    },
  },
```